python/frontend/cirrus/cirrus/GridSearch.py

```diff
@@ -62,6 +62,8 @@ def set_task_parameters(self, task, param_base=None, hyper_vars=[], hyper_params
             modified_config['ps_ip_private'] = machines[index][1]
             index = (index + 1) % num_machines
             base_port += 2
+
+            print(modified_config)
```

Comment:
I'm not sure if you actually want to print stuff to the command line. There is a logging library